jest.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  coverageDirectory: 'coverage',
+  setupFiles: ['<rootDir>/enzyme.config.js'],
+  testPathIgnorePatterns: ['/node_modules/'],
+  transformIgnorePatterns: ['<rootDir>/node_modules/', '<rootDir>/assets/img'],
+  moduleNameMapper: {
+    '.+\\.(css|scss)$': 'identity-obj-proxy',
```

Comment:
@chekwas88  could you tell me the purpose of this line? I'm wondering why it includes extensions like **`.css|scss`** and yet we don't write such files in React Native. I think you should leverage the following [expo doc](https://docs.expo.io/versions/latest/guides/testing-with-jest/) since you're using **Expo **.

Kindly, consider checking out [this](https://jestjs.io/docs/en/tutorial-react-native) too.